### DISCUSSION TITLE
似乎mmdet的即插即用plugin内并没有注意力机制的模块，请问如何添加注意力机制模块到网络中呢？

### DISCUSSION THREAD

**Discussion by ShaneCan** [open-mmlab/mmdetection#9025] [Q&A] [answered]
似乎mmdet的即插即用plugin内并没有注意力机制的模块，请问如何添加注意力机制模块到网络中呢？是不是需要调用其他库的模块呢？

**Comment by mm-assistant** [comment]
We recommend using English or English & Chinese for issues so that we could have broader discussion.

**Comment by ZwwWayne** [comment, marked as answer]
See examples in empirical attention in https://github.com/wanghonglie/mmdetection/tree/master/configs/empirical_attention
You can also insert non-local blocks.

**Reply by ShaneCan** [reply]
https://github.com/wanghonglie/mmdetection/blob/9d3e162459590eee4cfc891218dfbb5878378842/configs/empirical_attention/faster_rcnn_r50_fpn_attention_1111_dcn_1x_coco.py#L4

could i add this plugin in yolox config to achieve same function？In addition, i wanna know which .py defined this 'GeneralizedAttention', thx for response.
